packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -59,6 +59,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     copyPaste?: boolean;
     readOnly?: boolean;
     filterProp?: boolean;
+    renderToolbar?: boolean;
```

Comment:
other options also don't have a render prefix, why not just `toolbar`?

Reply:
I thought `renderToolbar` would be more telling of what the option does. I will rename it to match the others